tools/ci-scripts/circleci-build-catalog-next.bash

```diff
@@ -24,10 +23,9 @@ pip install --upgrade pip
 pip install setuptools -U
 pip install -r full_requirements.txt
 
-
-
 # extra pip
 pip install flask_debugtoolbar
+pip install google_compute_engine
```

Comment:
Can you add a comment on why this is necessary?

Reply:
Is necessary because for `boto` to use S3 in geodatagov commands @adborden.
I moved to poetry file and added a comment [here](required to boto plugin to upload to S3 in some internal commands)